t/udpfw.c

```diff
@@ -183,6 +190,25 @@ static int read_queue(struct queue_t *q, struct connection_t *conn, int64_t now)
                             conn != NULL ? NULL : &sslen)) <= 0)
         return 0;
 
+    packet_num++;
+
+    /* check if packet should be dropped */
+    if (q->num_drops > 0) {
+        int drop = 0;
+        if (packet_num > q->drops[q->num_drops])
```

Comment:
Do we have off-by-one error here?

Reply:
I don't think so. The last drop will be q->drops[q->num_drops], so this resets if we're done with the last drop.

Reply:
Ah sorry for the confusion. I mean that you might want to do `q->drops[q->num_drops - 1]`.